src/App.tsx

```diff
@@ -14,11 +15,67 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    selectedGood: 'Jam',
+  };
+
+  render(): React.ReactNode {
+    const { selectedGood } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          {selectedGood}
+          {' '}
+          is selected
```

Comment:
In case no good is selected, this msg will be "No goods is selected", it must be "No goods selected" - it's grammatically correct.